src/main/scala/com/yotpo/metorikku/output/writers/elasticsearch/ElasticsearchOutputWriter.scala

```diff
@@ -0,0 +1,46 @@
+package com.yotpo.metorikku.output.writers.elasticsearch
+
+import com.yotpo.metorikku.configuration.job.output.Elasticsearch
+import com.yotpo.metorikku.output.Writer
+import org.apache.log4j.LogManager
+import org.apache.spark.sql.{DataFrame, SaveMode}
+
+
+class ElasticsearchOutputWriter(props: Map[String, String], elasticsearchOutputConf: Option[Elasticsearch]) extends Writer {
+
+  case class ElasticsearchOutputProperties(saveMode: SaveMode, resource: String, mappingId: String, writeOperation: String,
+                                           port: Int, indexAutoCreate: Boolean, nodesWanOnly: Boolean)
+
+  val log = LogManager.getLogger(this.getClass)
+
+  val saveMode: SaveMode = SaveMode.valueOf(props.getOrElse("saveMode", "Overwrite"))
```

Comment:
I'm wondering if maybe append should be the default, like in all other writers

Reply:
Sure, I'll conform to the default